php/WP_CLI/SearchReplacer.php

```diff
@@ -92,5 +93,13 @@ private function _run( $data, $serialised, $recursion_level = 0, &$visited_data
 
 		return $data;
 	}
+
+	private function str_replace( $from, $to, $data ) {
```

Comment:
Yeah, this could definitely get pulled into `_run`. Pulling it out into a function was my knee-jerk reaction to nesting ifs inside ifs, but it definitely still reads.